### ISSUE TITLE
Review DualOwnableUpgradeable.sol implementation

### ISSUE THREAD

**Issue by 0xTranqui** [public-assembly/AssemblyPress#55]
tldr implemented new ownership logic for both ERC721 + ERC1155 factories (described in this [forum post](https://forum.public---assembly.com/t/new-ownership-implementation-for-assemblypress/396)) so that we can have a situation where both a trusted core team (ie the ppl working on the protocol) + PA as a whole can both push upgrades to the factory proxies, with PA having the ability to revoke the core team's access at any point. Link to [impl](https://github.com/public-assembly/AssemblyPress/blob/main/src/core/utils/ownable/dual/DualOwnableUpgradeable.sol) that is used/tested for the exact same in both 721/1155.

The [test](https://github.com/public-assembly/AssemblyPress/blob/main/test/ERC721/ERC721PressFactory.t.sol) coverage is sparse so probably worth improving this + just getting some eyes on it overall.
